android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -260,13 +274,17 @@ private void setUpAccountList() {
         }
     }
 
+    private void initState() {
+        pendingTxHelper = new PendingTxHelper(getEthTxService(), accountInfos, false, null);
```

Comment:
the same as above, `accountInfos` could be null, has to be moved `accountInfos = keyringInfo.accountInfos;` and not clear why do we need `PendingTxHelper` without calling `fetchTransactions`